hack/pin-update.sh

```
@@ -0,0 +1,50 @@
+
+# Get installed CSV from current subscription
+CSV=$(oc get subscription $MTV_SUBSCRIPTION -n $MTV_NAMESPACE -o json | jq -r '.status.installedCSV')
+# Remove the current subscription
+oc delete subscription -n $MTV_NAMESPACE $MTV_SUBSCRIPTION
+# Remove the current CSV
+oc delete csv -n $MTV_NAMESPACE $CSV
+# Remove the current Operator CR
+oc delete operators -n $MTV_NAMESPACE "$MTV_SUBSCRIPTION.$MTV_NAMESPACE"
+
+# Create a new subscription with patched version
+echo """apiVersion: operators.coreos.com/v1alpha1
```

Shellcheck (0.10.0)

[warning] Did you forget to close this double quoted string?

(SC1078)